petab/core.py

```diff
@@ -911,6 +907,18 @@ def append_overrides(overrides):
         append_overrides(
             split_parameter_replacement_list(row.noiseParameters))
 
+    # Append parameter overrides from condition table
+    condition_parameters = list(
+        set(condition_df.columns.values.tolist()) - {'conditionId',
+                                                     'conditionName'})
+    for overridee in condition_parameters:
```

Comment:
kinda hard to read. can you add some docs?

Reply:
Hm. Line 910? Which part do you mean?

Reply:
in particular what this invert stuff does